### PR TITLE
Update DG diagrams and links

### PR DESCRIPTION
**Pull request by clydelhui**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -689,6 +702,67 @@ The Sequence Diagram below illustrates the interactions within the Logic compone
...
+4. The `ImportCommand` instance is then returned to the `LogicManager`
+5. The `LogicManager` then executes the `ImportCommand` instance which obtains the data from the file (as a `ReadonlyAddressBook`) via the `Importer` API call
+6. A new `Person` is then created via the `createNewImportPerson()` call
+7. The `Person` is added to the `Model`
+8. Execution of `ImportCommand` results in a `CommandResult` created and returned back to the `LogicManager`, completing the execution.
+
+#### Design consideration
+
+**Aspect: Importing multiple persons**
+* **Alternative 1:** Export multiple `Person` in one user command.
```
Comment (lyndonlim27) [inline docs/DeveloperGuide.md:749]: Should be import right?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -689,6 +702,67 @@ The Sequence Diagram below illustrates the interactions within the Logic compone
...
+    * Pros:
+        * Users can import multiple persons at once instead of importing each person one at a time.
+    * Cons:
+        * More bug-prone as unable to verify the source of import files, import files could possibly be corrupted.
+
+* **[Current implementation] Alternative 2:** Only allow one person to be exported in one user command.
```
Comment (lyndonlim27) [inline docs/DeveloperGuide.md:755]: import also

File: docs/images/ComponentManagers.png
Comment (lyndonlim27) [inline docs/images/ComponentManagers.png:1]: is something suppose to change here?

File: docs/images/ImporterClassDiagram.png
Comment (lyndonlim27) [inline docs/images/ImporterClassDiagram.png:1]: the arrow from importcommand to importer should be blue

File: docs/images/LogicClassDiagram.png
Comment (lyndonlim27) [inline docs/images/LogicClassDiagram.png:1]: The box style of model is different from storage and importer. Not sure if it matters

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (clydelhui) [reply docs/DeveloperGuide.md:749]: Modified! Thanks!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (clydelhui) [reply docs/DeveloperGuide.md:755]: Also modified! Thanks again!

File: docs/images/ComponentManagers.png
Comment (clydelhui) [reply docs/images/ComponentManagers.png:1]: Thought about it but `Importer` does not communicate with any classes in logic except `ImportCommand` so I left it as it

File: docs/images/ImporterClassDiagram.png
Comment (clydelhui) [reply docs/images/ImporterClassDiagram.png:1]: Modified! Thanks!

File: docs/images/LogicClassDiagram.png
Comment (clydelhui) [reply docs/images/LogicClassDiagram.png:1]: Modified! Thanks

**Comment by lyndonlim27** [comment]
LGTM!

**Pull request merged**